### PR TITLE
app-arch/gzip: Various improvements

### PR DESCRIPTION
**Pull request by Whissi**
Various improvements:

- Migrate to EAPI=7
- Get rid of unusual USE=pic -- NO_ASM has no effect on most architectures so we should always set it. That's also Debian/Ubuntu and RHEL default.
- Set `UNALIGNED_OK` which should decrease (de)compression time by ~50%. Default in Debian/Ubuntu.

### CODE REVIEW

File: app-arch/gzip/gzip-1.10-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+EAPI=7
+
+inherit flag-o-matic
+
+DESCRIPTION="Standard GNU compressor"
+HOMEPAGE="https://www.gnu.org/software/gzip/"
+SRC_URI="mirror://gnu/gzip/${P}.tar.xz
+	mirror://gnu-alpha/gzip/${P}.tar.xz
+	mirror://gentoo/${P}.tar.xz"
```
Comment (thesamesam) [inline app-arch/gzip/gzip-1.10-r1.ebuild:12]: Is the mirror://gentoo a historical artefact here?

File: app-arch/gzip/gzip-1.10-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+LICENSE="GPL-3"
```
Comment (thesamesam) [inline app-arch/gzip/gzip-1.10-r1.ebuild:14]: GPL-3+

File: app-arch/gzip/gzip-1.10-r1.ebuild (hunk 1 above)
Comment (Whissi) [reply app-arch/gzip/gzip-1.10-r1.ebuild:12]: In the end it is, yes. Will get rid of it.

File: app-arch/gzip/gzip-1.10-r1.ebuild (hunk 2 above)
Comment (Whissi) [reply app-arch/gzip/gzip-1.10-r1.ebuild]: Thanks.

**Comment by Polynomial-C** [comment]
:+1:

**Comment by Polynomial-C** [comment]
:+1:

**Comment by ajakk** [comment]
Seems obsoleted and nacked by soap

**Pull request closed without merging**